vcr/vcr.go

```diff
@@ -224,8 +227,62 @@ func (c *vcr) Resolve(ID string) (did.VerifiableCredential, error) {
 	panic("implement me")
 }
 
-func (c *vcr) Verify(vc did.VerifiableCredential, credentialSubject interface{}, at time.Time) (bool, error) {
-	panic("implement me")
+func (c *vcr) Verify(vc did.VerifiableCredential, at time.Time) error {
+	// it must have valid content
+	validator, _ := credential.FindValidatorAndBuilder(vc)
+	if validator == nil {
```

Comment:
I think `Issue()` checks both validator and builder for `nil`? Let's decide on one style

Reply:
Only the validator is required here so no need to check the builder.